extensions/nats/README.md

```diff
@@ -47,9 +47,6 @@ path = "/path/to/build"
 Configure the extension through the Grafbase Gateway configuration file:
 
 ```toml
-[extensions.nats]
-path = "/path/to/build"
```

Comment:
I think this is still necessary, in this section (bulid from source).

Reply:
this is the Configuration section, not build?

Reply:
true, please ignore the comment then